apis/v1alpha1/httproute_types.go

```diff
@@ -392,20 +392,23 @@ const (
 	// HTTPRouteFilterRequestHeaderModifier can be used to add or remove an HTTP
 	// header from an HTTP request before it is sent to the upstream target.
 	//
-	// Support: Core
+	// Support in HTTPRouteRule: Core
+	// Support in HTTPRouteForwardTo: Extended
```

Comment:
Why is this filter considered "Extended" in `HTTPRouteForwardTo` and not `HTTPRouteRule`?

Reply:
Yeah this is confusing. I chose "extended" based on our current definitions in the API Concepts doc:

```
* **CORE** features will be portable and we expect that there is a
  reasonable roadmap for ALL implementations towards support of APIs
  in this category.
* **EXTENDED** features are those that are portable but not
  universally supported across implementations. Those implementations
  that support the feature will have the same behavior and
  semantics. It is expected that some number of EXTENDED features will
  eventually migrate into the CORE. EXTENDED features will be part of
  the API types and schema.
* **CUSTOM** features are those that are not portable and are
  vendor-specific. CUSTOM features will not have API types and schema
  except via generic extension points.
```

In this case I think something like a request header modifier would be implemented the same way by all implementations that could support the feature on ForwardTo. That makes it portable and thus not custom. On the other hand, I don't think there is a reasonable roadmap for ALL implementations to support filters at this level, so it also can't be core. That leaves extended.